pkg/pullsync/pullstorage/pullstorage.go

```diff
@@ -43,79 +45,97 @@ type Storer interface {
 // ps wraps storage.Storer.
 type ps struct {
 	storage.Storer
+	intervalFlight *singleflight.Group
```

Comment:
Type ps is used with pointer semantics, it is not required to have the Group as pointer as well.